src/OpenDebugAD7/cppdbg.ad7Engine.json

```diff
@@ -7,9 +7,15 @@
   "exceptionSettings": {
     "categories": [
       {
-        "name": "CppExceptionCategory",
+        "name": "C++ Exceptions",
         "id": "3A12D0B7-C26C-11D0-B442-00A0244A1DD2"
       }
+    ],
+    "exceptionBreakpointFilters": [
+      {
+        "label": "All Exceptions",
```

Comment:
>Exceptions [](http://example.com/codeflow?start=22&length=10)

I wonder if we should name this `All C++ Exceptions` since someday we might support signals?